db/schema.rb

```diff
@@ -11,13 +11,13 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20150609174024) do
+ActiveRecord::Schema.define(version: 20150609193512) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
   enable_extension "uuid-ossp"
 
-  create_table "artists", id: :uuid, default: "uuid_generate_v4()", force: :cascade do |t|
+  create_table "artists", id: :uuid, default: nil, force: :cascade do |t|
```

Comment:
Hmm, I think maybe by enabling the UUID extension for postgres, it set the default for IDs on the artists table to nil.  Try rolling back that migration, removing the extension, and migrating again and see if this converts back to the default `uuid_generate_v4()`